src/renderer/components/SchemaDesign/store/actions.js

```diff
@@ -201,7 +203,7 @@ export default {
     const node = state.visFacade.getNode(state.selectedNodes[0].id);
 
     const ownerConcept = await graknTx.getSchemaConcept(node.label);
-    const edges = await CDB.getTypeAttributeEdges(ownerConcept);
+    const edges = await CDB.getTypeEdges(ownerConcept, state.visFacade.getAllNodes().map(n => n.id));
```

Comment:
shouldn't you be filtering by attribute type edges?

Reply:
what we're doing here is constructing the edges that point out of the `ownerConcept` to any node that's already visualised. 

it may be the `get` in the method name `getTypeEdges` that's misled you?

Reply:
how about `getSchemaEdges()`?  `type` confused me